### ISSUE TITLE
Standardize rule names

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [confuser/graphql-constraint-directive#14]
This package is currently in fact dealing with two things:

* the `value` of field
* the `length` of field

The names of rules for these things can be improved (standardized).

What's wrong with current names? They are not intuitive. E.g. `min` and `minLength` are `greater than` or `greater than or equal to`?

I really like how [Prisma](https://github.com/prisma/prisma) auto-generates names by postfixes:

`<thing>_lt`
`<thing>_lte`
`<thing>_gt`
`<thing>_gte`

`<thing>_contains`
`<thing>_not_contains`
`<thing>_starts_with`
`<thing>_not_starts_with`
`<thing>_ends_with`
`<thing>_not_ends_with`

`<thing>_in`
`<thing>_not_in`

`<thing>_every`
`<thing>_some`
`<thing>_none`

etc.

In other words, I propose to rename `min` to `value_gte`, `minLength` to `length_gte`, `notContains` to `value_not_contains`, etc.

**Comment by confuser** [comment]
Thanks for the suggestion, however I'd prefer to stick with camel case notations
